catalogue/webapp/components/ItemRequestModal/ItemRequestModal.tsx

```diff
@@ -114,8 +114,8 @@ const RequestDialog: FC<RequestDialogProps> = ({
   function handleConfirmRequest(event: FormEvent<HTMLFormElement>) {
     event.preventDefault();
 
-    if (pickUpDate) {
-      confirmRequest(pickUpDate);
+    if (!enablePickUpDate || (enablePickUpDate && pickUpDate)) {
```

Comment:
I think this is just equivalent to
```suggestion
    if (!enablePickUpDate || pickUpDate) {
```